lib/elixir_internal_certificate/scrapper/schemas/user_search.ex

```diff
@@ -1,6 +1,8 @@
 defmodule ElixirInternalCertificate.Scrapper.Schemas.UserSearch do
   use Ecto.Schema
 
+  import Ecto.Changeset
```

Comment:
Lets import what we need only 
```suggestion
  import Ecto.Changeset, only: [change: 2]
```